pkg/core/resources/store/options.go

```diff
@@ -211,6 +211,10 @@ func ListOrdered() ListOptionsFunc {
 	}
 }
 
+func (l *ListOptions) IsCacheable() bool {
+	return l.FilterFunc == nil
```

Comment:
I think if you add pages it's also not cacheable no?

Reply:
True, added this and `NamePrefix`